### PR TITLE
feat: upgrade to Lighthouse 10.1.0

### LINKED ISSUE #886: Chrome v111 with lhci/cli@v0.11.0: Cannot create new tab, and no tabs already open
Documented in #https://github.com/GoogleChrome/lighthouse/issues/14885 . Sounds like upgrading the Lighthouse version should resolve it.

### LINKED ISSUE #891: Upgrade lighthouse version from 9.6.8 to 10.0.2
I encountered some issues with lighthouse ci (lighthouse version 9.6.8) that are fixed with the version 10.0.2.
Issue link: [https://github.com/GoogleChrome/lighthouse-ci/issues/887](https://github.com/GoogleChrome/lighthouse-ci/issues/887)

### PR DESCRIPTION
**Pull request by adamraine**
fixes #891
fixes #886

### CODE REVIEW

File: packages/cli/test/collect.test.js [hunk 1]
```diff
@@ -100,12 +100,12 @@ describe('collect', () => {
         // Check server started and lighthouse ran.
         const cleanStartCommand = cleanStdOutput(startCommand);
         expect(stdout).toMatchInlineSnapshot(`
-        "Started a web server with "${cleanStartCommand}"...
-        Running Lighthouse 1 time(s) on http://localhost:XXXX/
-        Run #1...done.
-        Done running Lighthouse!
-        "
-      `);
+          "Started a web server with "SERVER_START_PORT=XXXX SERVER_START_MESSAGE='Running server' node /Users/asraine/src/lighthouse-ci/packages/cli/test/fixtures/autorun-start-server/autorun-server.js"...
```
Comment (paulirish) [inline packages/cli/test/collect.test.js:103]: i guess this got tweaked by -U but should be ~reverted

File: packages/cli/test/collect.test.js [hunk 2]
```diff
@@ -141,14 +141,14 @@ describe('collect', () => {
...
         expect(stdout).toMatchInlineSnapshot(`
-        "Started a web server with "${cleanStartCommand}"...
-        WARNING: Timed out waiting for the server to start listening.
-                 Ensure the server prints a pattern that matches /listen|ready/i when it is ready.
-        Running Lighthouse 1 time(s) on http://localhost:XXXX/
-        Run #1...done.
-        Done running Lighthouse!
-        "
-      `);
+          "Started a web server with "SERVER_START_PORT=XXXX SERVER_START_MESSAGE='Running server' node /Users/asraine/src/lighthouse-ci/packages/cli/test/fixtures/autorun-start-server/autorun-server.js"...
```
Comment (paulirish) [inline packages/cli/test/collect.test.js:144]: same

File: packages/server/test/ui/__image_snapshots__/storybook-test-jsx-image-storyshots-build-view-audit-detail-pane-version-1010-1-snap.png
Comment (paulirish) [inline packages/server/test/ui/__image_snapshots__/storybook-test-jsx-image-storyshots-build-view-audit-detail-pane-version-1010-1-snap.png:1]: something weird here

File: packages/server/test/ui/__image_snapshots__/storybook-test-jsx-image-storyshots-build-view-lhr-comparison-version-1010-1-snap.png
Comment (paulirish) [inline packages/server/test/ui/__image_snapshots__/storybook-test-jsx-image-storyshots-build-view-lhr-comparison-version-1010-1-snap.png:1]: something weird here

**Review by paulirish — CHANGES_REQUESTED** [review]
a couple things i spotted
but overall lookin great

File: packages/server/test/ui/__image_snapshots__/storybook-test-jsx-image-storyshots-build-view-lhr-comparison-version-1010-1-snap.png
Comment (paulirish) [inline packages/server/test/ui/__image_snapshots__/storybook-test-jsx-image-storyshots-build-view-lhr-comparison-version-1010-1-snap.png:1]: this dude?

**Review by paulirish — APPROVED** [review]

**Comment by chaimabg** [comment]
Hello @adamraine, I wanted to check if this Pull Request will be included in an upcoming release ?

**Comment by adamraine** [comment]
Yes, we are hoping to do a breaking release today

**Pull request merged**